.github/workflows/audit.yml

```diff
@@ -13,6 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions-rs/audit-check@v1
+      - uses: EmbarkStudios/cargo-deny-action@v1
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Nice to remove this. Why was it needed before?

Reply:
Not sure. Don't quote me on this, but I think it was just the old style of passing the token down, and github added a new implicit style for actions, that doesn't require this boilerplate. So it's not actually a real improvement. But it's a read-only token anyway here, so it doesn't matter.